public/static/docs/tutorials/versioning.md

```diff
@@ -15,8 +15,8 @@ to build a powerful image classifier using a pretty small dataset.
 
 We first train a classifier model using 1000 labeled images, then we double the
 number of images (2000) and retrain our model. We capture both datasets and
-classifier results and show how to use `dvc checkout` along with `git checkout`
-to switch between different versions.
+classifier results and show how to use `dvc checkout` to switch between data or
```

Comment:
it's at least data and/or model. Still don't see a good reason for this change tbh (probably missed some explanation, sorry - it's near to impossible to track everything in this PR :) )

Reply:
I think the reason is that this paragraph's intention is to introduce the steps that the tutorial will follow, and their purpose. The purpose is not to show how to use `git checkout`, but `dvc checkout`.

And no, I don't think I had explained this before. Sorry if sometimes there are changes that seem totally unrelated but it's things I stumble upon in the same region of text I'm changing for some other reason and it would be 3 times harder to somehow make a note or save the change in a separate branch or stash to apply properly in a regular updates PRs. I try to avoid these as much as possible but they sometimes happen.

Reply:
> I try to avoid these as much as possible but they sometimes happen.

I totally understand this. I was doing docs myself a lot.

> which implies git checkout so no need to mention

this is not exactly true, you can do `dvc checkout` to relink for example? or after clone or if you deleted something.

regarding the version term here.

Again not 100% sure but it feels that in this case its like switching between different versions of a model or dataset. and version is appropriate. Even title includes versioning.

Reply:
> it's at least data and/or model

Did you just meant that we should have `and/or` instead of `or`? Actually, reviewing the examples in the corresponding section further down, just "and" is most appropriate here. Changing...

Reply:
>  "and" is most appropriate here. Changing...

Probably not - you can switch a particular file. I hope this tutorial has that example.

Reply:
I think in common language "and" is not exclusive like in discrete logic. But anyway, OK, using "and/or" to be extra clear. And it does (have the particular file example).

Reply:
sure. when I was mentioning and/or it was not a suggestion of specific phrase to use (I usually emphasize if it's a specific suggestion). `and/or` is the worst option I would say :) I was just trying to make a point about the problem. `and` was better - let's use it, unless we can come up with a better idea.

Reply:
OK. How about "Switching between workspace versions"? Updating in #988